docs/content/operations/other-hadoop.md

```diff
@@ -51,11 +56,23 @@ hadoop-dependencies/
     ..... lots of jars
 ```
 
-As you can see, under ```hadoop-client```, there are two sub-directories, each denotes a version of ```hadoop-client```. During runtime, Druid will look for these directories and load appropriate ```hadoop-client``` based on `hadoopDependencyCoordinates` passed to the [Hadoop Index Task](../ingestion/tasks.html).
+As you can see, under ```hadoop-client```, there are two sub-directories, each denotes a version of ```hadoop-client```. 
+
+**Tell Druid what version of Hadoop to load**
+
+Use `hadoopDependencyCoordinates` in [Hadoop Index Task](../ingestion/batch-ingestion.html) to specify the Hadoop dependencies you want Druid to load.
+
+For example, in your Hadoop Index Task spec file, you have
+
+`"hadoopDependencyCoordinates": ["org.apache.hadoop:hadoop-client:2.4.0"]`
```

Comment:
it seems odd to have "org.apache.hadoop" here when Druid isn't actually going to the maven repository, is it?  What is says below is that Druid is just looking for a directory hadoop-client/2.4.0.